modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc

```diff
@@ -214,7 +216,7 @@ If the store is seeded with `seedURI` or `existingDataSeedServer`, or if the com
 
 | `seedRestoreUntil`
 | Datetime or transaction id. E.g. `datetime("2025-01-01T12:15:00.000+0100")` or `123456`
-| 
+|
 If you are passing a `seedURI` that leads to a backup chain, including differential backups, you can choose to not apply all the transactions in the differential backups.
 To seed up to a specific date, specify a `datetime`.
 This will seed the database with transactions committed before the provided timestamp.
```

Comment:
See line 226 about the `seedSourceDatabase`.This option is introduced in Cypher 25. Bu do we need also to add a label `Introduced in 2025.06`? I suppose in the future we'll have many things introduced in Cypher25 but in different Neo4j versions.

Reply:
Yes, I think it makes senses.